projects/storefrontlib/cms-components/product/product-tabs/product-reviews/product-reviews.component.ts

```diff
@@ -99,7 +105,9 @@ export class ProductReviewsComponent {
       alias: reviewFormControls.reviewerName.value,
     };
 
-    this.reviewService.add(product.code, review);
+    if (product.code) {
```

Comment:
Should we throw an error of some kind or just silently not call the function?

Reply:
if there is product, it must have code. I can see that in most cases, we just use default ` ?? ''` for it. I can do it as well.